src/components/Login/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { forceLogout } from '../../actions/auth';
+import * as nulApi from '../../services/nul-api.js';
+
+class Login extends Component {
+  render() {
+    const authToken = this.props.authToken || '';
+
+    const loginClick = () => nulApi.login();
+    const logoutClick = () => this.props.forceLogout();
+
+    var loginText;
+    if (authToken === '') {
+      loginText = (
+        <a href={nulApi.loginLink()} onClick={loginClick}>
+          Sign in
+        </a>
+      );
+    } else {
+      loginText = (
+        <li className="resources">
+          {nulApi.currentUser()}
+          <span className="dropdown-arrow" />
+          <ul>
+            <li>
+              <button onClick={logoutClick}>
+                <strong>SIGN OUT</strong>
+              </button>
+            </li>
+          </ul>
+        </li>
+      );
+    }
+
+    return <span>{loginText}</span>;
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    authToken: state.auth.token
+  };
+};
+
+const mapDispatchToProps = dispatch => ({
+  forceLogout: () => dispatch(forceLogout())
+});
+
```

Comment:
Somewhere here if you wanted (in the future) to include `PropTypes`, it's a React library which helps show a components incoming `props` so random team members know what data they have to work with.  It's used pretty heavily throughout the app if you wanted an example.  No big deal now, but a pattern to use.